astrosat_users/views/views_api_users.py

```diff
@@ -1,10 +1,18 @@
+from django.core.exceptions import ValidationError
+from django.db.models import Count
+
 from rest_framework.permissions import (
     IsAuthenticated,
     IsAdminUser,
     BasePermission,
     SAFE_METHODS,
 )
 from rest_framework import mixins, viewsets
+from rest_framework.exceptions import APIException
+
+from django_filters import rest_framework as filters
+
+from astrosat.views import BetterBooleanFilter, BetterBooleanFilterField
```

Comment:
`BetterBooleanField`, not sure what that really means, is there a better descriptive name we can think of?

Reply:
I still feel you've added a workaround for clients sending you poor info, but I do understand your explanation and if you still think it appropriate, I'm fine with that.